Cartfile

```diff
@@ -2,7 +2,7 @@ binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.6
 github "mapbox/MapboxDirections.swift" ~> 0.12
 github "mapbox/turf-swift" ~> 0.0.3
 github "rs/SDWebImage" ~> 4.1
-github "Project-OSRM/osrm-text-instructions.swift" ~> 0.5
+github "Project-OSRM/osrm-text-instructions.swift" "f60d5bc175794aee6c7199d60548820e0fb23097"
```

Comment:
Will change when https://github.com/Project-OSRM/osrm-text-instructions.swift/pull/50 is released.